osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -20,18 +20,24 @@ public class Speed : Skill
         protected override double StrainValueOf(OsuDifficultyHitObject current)
         {
             double distance = current.Distance;
+            double speedBonus = 1.0;
+            
+            if(current.DeltaTime < 68) // 68 = 220 BPM 1/4th snapping in MS. 
```

Comment:
220bpm at 1/4 isn't exactly 68ms, might want to make it an algorithm

doesnt really matter though, i guess